packages/benchkit-backend/resources/execute.requests.http

```diff
@@ -0,0 +1,388 @@
+### Single query with default params
+PUT http://localhost:8080/workload/
+Content-Type: application/json
+
+{
+    "queries": [{
+        "text": "RETURN 1"
+    }]
+}
+
+### Mutiple queries with default params
+PUT http://localhost:8080/workload/
+Content-Type: application/json
+
+{
+    "queries": [{
+        "text": "RETURN 1"
+    },
+    {
+        "text": "RETURN 1"
+    },
+    {
+        "text": "RETURN 1"
+    }]
+}
+
+### Mutiple queries with in sequence params
+PUT http://localhost:8080/workload/
+Content-Type: application/json
+
+{
+    "mode": "parallelSessions",
+    "queries": [{
+        "text": "RETURN $a",
+        "parameters": {
+            "a": 1
+        }
+    },
+    {
+        
```

Comment:
Nit: black line without identifiable benefit.

(more of those in the file, won't comment again)